test/minikube/001-kube.bats

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bats
+#
+# Tests of podman kube commands with minikube
+#
+
+load ../system/helpers.bash
```

Comment:
Recommendation: set up your own `helpers.bash` in this subdirectory. From it, do the `load ../whatever` and the first-and-last-test thing. (Unless you plan to only ever have `001.bats`, which seems unlikely)

Reply:
fixed